drivers/scheduler/k8s/specs/fio-fa-cloud-drives/fio.yaml

```diff
@@ -61,7 +61,7 @@ spec:
       - ReadWriteOnce
       resources:
         requests:
-          storage: 200Gi
+          storage: 10Gi
```

Comment:
Better to use yaml template to set the size at run time. Basically when using the same spec in system test, it would be good to have bigger size ..

`  {{ if .VolumeSize }}
  storage: "{{ .VolumeSize }}"
  {{ else }}
  storage: "10Gi"{{ end }}`

Reply:
Sounds good! Thank you, will fix